src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -36,15 +36,25 @@ public boolean contains(Person toCheck) {
         return internalList.stream().anyMatch(toCheck::isSamePerson);
     }
 
+    /**
+     * Gets a person by phone.
+     * The phone may not belong to any person.
```

Comment:
```suggestion
     * A phone belongs to only one person.
```

Reply:
For this it's true that the phone argument may not belong to any person before the search. That's why it returns an `Optional<Person>`